src/backend/assets/js/session.js

```diff
@@ -9,16 +9,10 @@ $(document).ready(function() {
   var sessionId = sessionElem.attr('id');
 
   function init() {
-    var temp;
-
-    if (localStorage.hasOwnProperty(eventName) === false) {
-      localStorage[eventName] = '{}';
-    }
-
-    temp = JSON.parse(localStorage[eventName]);
+    var temp = JSON.parse(localStorage[eventName] || '{}');
```

Comment:
I believe this should have an equivalent effect, but in one line.

If the event is missing from the storage (or an empty string) then this will initialise it to an empty object.

(I did the same elsewhere, simply for consistency.)